src/backend/marsha/core/utils/api_utils.py

```diff
@@ -49,3 +50,8 @@ def generate_hash(secret, message):
     return hmac.new(
         secret.encode("utf-8"), msg=message, digestmod=hashlib.sha256
     ).hexdigest()
+
+
+def generate_salted_hmac(secret, key):
+    """Generate a salted_hmac with secret and key"""
+    return salted_hmac(secret, key, hashlib.sha256).hexdigest()[::2]
```

Comment:
```suggestion
    return salted_hmac(secret, key, algorithm="sha256").hexdigest()[::2]
```
Are you sure using `hashlib.sha256` is working ? Should not be simply `sha256` ? Because in the `salted_hmac` function they check if the algo is a property of `haslib`.
And `algorithm` is not a positional argument in the function, so you should set this parameter